plugins/modules/mso_schema_site_anp_epg_useg_attribute.py

```diff
@@ -0,0 +1,264 @@
+            ["state", "query", ["schema", "template", "anp", "epg", "site"]],
+            ["state", "absent", ["schema", "template", "anp", "epg", "name", "site"]],
+            ["state", "present", ["schema", "template", "anp", "epg", "name", "attribute_type", "site"]],
+            ["useg_subnet", False, ["value"]],
+        ],
+    )
+
+    schema = module.params.get("schema")
+    site = module.params.get("site")
+    template = module.params.get("template").replace(" ", "")
+    anp = module.params.get("anp")
+    epg = module.params.get("epg")
+    name = module.params.get("name")
+    description = module.params.get("description")
+    attribute_type = module.params.get("attribute_type")
+    value = module.params.get("value")
+    operator = module.params.get("operator")
+    useg_subnet = module.params.get("useg_subnet")
+    state = module.params.get("state")
+    mso = MSOModule(module)
+
+    if state == "present":
+        if attribute_type in ["mac", "dns"] and value is None:
+            mso.fail_json(msg="Failed due to invalid 'value' and the attribute_type is: {0}.".format(attribute_type))
+        elif attribute_type not in ["mac", "dns", "ip"] and (value is None or operator is None):
+            mso.fail_json(msg="Failed due to invalid 'value' or 'operator' and the attribute_type is: {0}.".format(attribute_type))
+
+    mso_schema = MSOSchema(mso, schema, template)
+    mso_schema.set_template(template)
+    mso_schema.set_template_anp(anp)
+    mso_schema.set_template_anp_epg(epg)
+
+    mso_schema.set_site(template, site)
+    mso_schema.set_site_anp(anp)
+    mso_schema.set_site_anp_epg(epg)
+
+    if mso_schema.schema_objects["template_anp_epg"].details.get("uSegEpg"):
+        mso_schema.set_site_anp_epg_useg_attr(name, fail_module=False)
+        if mso_schema.schema_objects["site_anp_epg_useg_attribute"] is not None:
+            site_useg_attr_path = "/sites/{0}-{1}/anps/{2}/epgs/{3}/uSegAttrs/{4}".format(
+                mso_schema.schema_objects["site"].details.get("siteId"), template, anp, epg, mso_schema.schema_objects["site_anp_epg_useg_attribute"].index
+            )
+            mso.existing = mso_schema.schema_objects["site_anp_epg_useg_attribute"].details
+    else:
+        mso.fail_json(msg="{0}: is not a valid uSeg EPG.".format(epg))
+
+    if state == "query":
+        if name is None:
+            mso.existing = mso_schema.schema_objects["site_anp_epg"].details.get("uSegAttrs")
+        elif not mso.existing:
+            mso.fail_json(msg="The uSeg Attribute: {0} not found.".format(name))
+        mso.exit_json()
+
+    site_useg_attrs_path = "/sites/{0}-{1}/anps/{2}/epgs/{3}/uSegAttrs".format(mso_schema.schema_objects["site"].details.get("siteId"), template, anp, epg)
+    ops = []
+    mso.previous = mso.existing
+
+    if state == "absent":
+        if mso.existing:
+            mso.existing = {}
+            ops.append(dict(op="remove", path=site_useg_attr_path))
+    elif state == "present":
```

Comment:
to avoid partial in our coverage
```suggestion
    if state == "present":
```